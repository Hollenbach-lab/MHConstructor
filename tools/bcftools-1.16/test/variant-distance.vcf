##fileformat=VCFv4.2
##reference=file://some/path/human_g1k_v37.fasta
##contig=<ID=1,length=2147483647>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	101	.	T	A	.	.	.
1	101	.	T	C	.	.	.
1	104	.	T	A	.	.	.
1	104	.	T	C	.	.	.
1	108	.	T	A	.	.	.
1	108	.	T	C	.	.	.
1	115	.	T	A	.	.	.
