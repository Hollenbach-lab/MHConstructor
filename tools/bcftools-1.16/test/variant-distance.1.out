##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##reference=file://some/path/human_g1k_v37.fasta
##contig=<ID=1,length=2147483647>
##INFO=<ID=DIST,Number=1,Type=Integer,Description="Distance to the  nearest variant">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	101	.	T	A	.	.	DIST=3
1	101	.	T	C	.	.	DIST=3
1	104	.	T	A	.	.	DIST=3
1	104	.	T	C	.	.	DIST=3
1	108	.	T	A	.	.	DIST=4
1	108	.	T	C	.	.	DIST=4
1	115	.	T	A	.	.	DIST=7
